<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGradConfig</class>
 <widget class="QDialog" name="DialogGradConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabGradDesc">
        <attribute name="title">
         <string>Grad Desc</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>171</width>
           <height>294</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Min eta</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="EditMinEta">
            <property name="maxLength">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>First jump</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="EditFirstJump">
            <property name="maxLength">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>k mult</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="Edit_k_mult">
            <property name="maxLength">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>k div</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="Edit_k_div">
            <property name="maxLength">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>eps</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="Edit_eps">
            <property name="maxLength">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>alpha</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="EditAlpha">
            <property name="maxLength">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Callback freq</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Derivate</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="EditCallbackFreq"/>
          </item>
          <item row="7" column="1">
           <widget class="QCheckBox" name="chbFinMethod">
            <property name="text">
             <string>Fin method</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="lblMaxiters">
            <property name="text">
             <string>Max iters</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Max time, s</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="EditMaxIters"/>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="EditMaxTime"/>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="btnGradDescOpenFile">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>141</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Open Settings FIle...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="EditGradDescFile">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>431</width>
           <height>24</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="btnGradDescSaveAs">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>330</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Save As...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnGradDescSave">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>330</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabTargetFunc">
        <attribute name="title">
         <string>Target Func Global</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>170</width>
           <height>232</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Aarf</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="EditAarf"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>A2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="EditA2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>p</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="Edit_p"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>offX</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="Edit_offX"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>k_step_ot</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="Edit_k_step_ot"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>R_nodeOverlap</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="EditR_nodeOverlap"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Covered</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="chbIsUseCoveredFlag">
            <property name="text">
             <string>Use</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>LineBetweenPoints</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QCheckBox" name="chbIsUseLineBetweenTwoPoints">
            <property name="text">
             <string>Use</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>170</y>
           <width>171</width>
           <height>84</height>
          </rect>
         </property>
         <property name="title">
          <string>Signal Nodes Type</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="rbSignalSphere">
            <property name="text">
             <string>Shpere Signal Nodes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbSignalCone">
            <property name="text">
             <string>Cone Signal Nodes</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="btnTargetFuncGlobalSaveAs">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>340</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Save As...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnTargetFuncGlobalSave">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>340</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnTargetFuncGlobalOpenFile">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>141</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Open Settings FIle...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="EditTargetFuncGlobalFile">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>431</width>
           <height>24</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabTargetFuncSelector">
        <attribute name="title">
         <string>Func Selector</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>251</width>
           <height>141</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Target Func First Phase:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbTargetFuncFirstPhase"/>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Target Func Second Phase:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbTargetFuncSecondPhase"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tabRelief">
        <attribute name="title">
         <string>Relief</string>
        </attribute>
        <widget class="QGroupBox" name="groupBoxGridReliefSettings">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>151</width>
           <height>131</height>
          </rect>
         </property>
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QFormLayout" name="formLayout_GridReliefSettings">
          <item row="0" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>dx, meters</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>dy, meters</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>nDetails</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="EditGrid_dx">
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="EditGrid_dy">
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="EditGrid_nDetails">
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogGradConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogGradConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
